lib/core/failure/api_error_response.dart

```diff
@@ -0,0 +1,56 @@
+import 'dart:convert';
```

Comment:
remove this file, you may not need this file for error handling

Reply:
ApiErrorResponse class has been updated